### PR TITLE
chore: changing wording and trigger for id taken error

### PR DESCRIPTION
**Pull request by moliva**
# What? 
...

# Why? 
...

### CODE REVIEW

File: kernel/packages/shared/index.ts [hunk 1]
```diff
@@ -258,7 +261,11 @@ export async function initShared(): Promise<Session | undefined> {
 
       break
     } catch (e) {
-      if (e.message && e.message.startsWith('error establishing comms')) {
+      if (e.message && e.message.includes('is taken')) {
```
Comment (BrianAmadori) [inline kernel/packages/shared/index.ts]: Checking errors like this is very error prone. We should use type checks instead of this.

Comment (eordano) [reply kernel/packages/shared/index.ts]: Agreed! Do we have a list of these errors?

Comment (moliva) [reply kernel/packages/shared/index.ts]: Addressed! We do not have a list of them, but rather create the difference when trying to treat them separately.

**Comment by eordano** [comment]
Heads up @xpalaum this reverts the usability of "log in on multiple instances". We should be aware of this and notify Sam.

**Review by eordano — APPROVED** [review]

**Pull request merged**